### DISCUSSION TITLE
504 internal server error on teleport 13+ (nginx loadbalancer)

### DISCUSSION THREAD

**Discussion by Giannischri** [gravitational/teleport#48895] [Q&A]
I have nginx loadbalancer and cloudflare configuration...i am following the teleport 13+ guide for the values inside the teleport cluster helm chart....when users get their invite tokens and register they get a 504 internal server error immediately after registering.Some users have managed to get access while others (especially macos users) couldnt

**Comment by stevenGravy** [comment]
Greetings, Teleport 13 is no longer maintained and v14 is about to be EOL. Please try this on more recent versions and confirm if you are still seeing this.

**Comment by webvictim** [comment]
Do you have mixed IPv4 and IPv6 clients in your environment? I see this same issue when native IPv6 clients try to connect to my cluster (which is on IPv4 only) through Cloudflare.

What shows in the log files for the Teleport auth/proxy service when you see the 504 error?
